rust/repo/{{ cookiecutter.repo_name }}/README.md

```diff
@@ -1,17 +1,38 @@
+<!-- markdownlint-disable -->
```

Comment:
:x: The markdown lint disable seems suspect it's getting propagated to the rendered output.
I'm guessing that the root `.markdownlint-cli2.jsonc` needs to be updated for the new directory structure.
```json
  "ignores": [".github/pull_request_template.md", "target/", "rust-workspace/{{cookiecutter.repo_name }}/", "rust-crate/{{ cookiecutter.crate_sub_dir }}/"]   
```

Reply:
Fixed in the crate-target PR.